server/app-engine/static/graas.js

```diff
@@ -741,34 +742,33 @@ function positionCallback() {
 
 function handleGPSUpdate(position) {
     util.log('handleGPSUpdate()');
-    var lat, long, accuracy, posTimestamp, speed, heading;
-    var uuid = getUUID();
+    let uuid = getUUID();
 
     if (position) {
-        lat = position.coords.latitude;
-        long = position.coords.longitude;
-        accuracy = position.coords.accuracy;
-        posTimestamp = Math.round(position.timestamp / 1000);
-        speed = position.coords.speed;
-        heading = position.coords.heading;
+        let lat = position.coords.latitude;
```

Comment:
I don't think this works. These variables are now scoped for the `if` block, but are later accessed outside it.